src/draw/vg_lite/lv_draw_vg_lite.c

```diff
@@ -204,7 +226,7 @@ static int32_t draw_evaluate(lv_draw_unit_t * draw_unit, lv_draw_task_t * task)
     LV_UNUSED(draw_unit);
 
     switch(task->type) {
-        case LV_DRAW_TASK_TYPE_LABEL:
+        // case LV_DRAW_TASK_TYPE_LABEL:
```

Comment:
Accidentally left here commented (because of debugging), or it is really not needed? 
In latter case you could remove it, git history will have it anyway.